### PR TITLE
[TS-28]: Fix the Alert box and Navigation Area Overlapping Bug, Sidebar Moderator Box Overlapping Issue and Update Sidebar Navigation Links Name

### PR DESCRIPTION
**Pull request by hatemquliyev**
- add new sidebar links and update links name
- update some style codes
- fix alert box
- fix sidebar overlapping issue
- refactor codes

### CODE REVIEW

File: client/src/Admin/components/alert/Alert.tsx [hunk 1]
```diff
@@ -25,33 +24,30 @@ export interface IPropsType {
...
-                                <h6 className='m-0'>
-                                    {alertType}
-                                </h6>
-                                <p className='m-0'>
-                                    {message}
-                                </p>
-                            </div>
-                        </div>
+        {/* <div className='alert-box'> */}
+            <div ref={alertSection}
```
Comment (ilkinnezer1) [inline client/src/Admin/components/alert/Alert.tsx:32]: *alertBox

File: client/src/Admin/components/alert/Alert.tsx [hunk 2]
```diff
@@ -25,33 +24,30 @@ export interface IPropsType {
...
-                            <div className="alert_title-description px-3">
-                                <h6 className='m-0'>
-                                    {alertType}
-                                </h6>
-                                <p className='m-0'>
-                                    {message}
-                                </p>
-                            </div>
-                        </div>
+        {/* <div className='alert-box'> */}
```
Comment (ilkinnezer1) [inline client/src/Admin/components/alert/Alert.tsx:31]: What is that?

**Review by ilkinnezer1 — CHANGES_REQUESTED** [review]

File: client/src/Admin/components/alert/Alert.tsx [hunk 3]
```diff
@@ -25,33 +24,30 @@ export interface IPropsType {
...
+                            {alertType}
+                        </h6>
+                        <p className='m-0'>
+                            {message}
+                        </p>
                     </div>
                 </div>
             </div>
-        </div>
+            {/* </div> */}
```
Comment (ilkinnezer1) [inline client/src/Admin/components/alert/Alert.tsx:50]: Remove this if this is a comment

File: client/src/Admin/auth/LoginPage.scss [hunk 4]
```diff
@@ -1,9 +0,0 @@
-.login-page_container{
```
Comment (ilkinnezer1) [inline client/src/Admin/auth/LoginPage.scss:1]: If we don't need this file, please remove it

Comment (hatemquliyev) [reply client/src/Admin/auth/LoginPage.scss:1]: It has already been removed

**Pull request merged**